src/components/layout/SideBar.jsx

```diff
@@ -13,6 +13,12 @@ const { Sider } = Layout;
 
 const SideBar = props => {
 	const { context, dispatch } = useContext(Context);
+	
+	const [winWidth, setWinWidth] = useState(window.innerWidth);
+	function change(wid){ setWinWidth(wid) }
+	let headStyles = winWidth < 768 ? {fontSize: '1.15em'} : {fontSize: '2vw'}
```

Comment:
why not just use true/false for this winWidth state variable? It seems it is either a quarter of the screen or full screen, no in between?